README.md

```diff
@@ -532,6 +552,30 @@ Breaking changes are changes that could break a client that is relying on the Op
 [See some examples of breaking and non-breaking changes](breaking-changes.md).  
 Note: this is a Beta feature. Please report issues.
 
+### Breaking Changes Checks
+Am improved method for detecting breaking changes in specifications. It works differently form the original "Breaking changes" feature.
+
+There are multiple advantages using this feature in comparison with the original implementation:
```

Comment:
Instead of explaining the advantages, which may interest existing users but not new users, I suggest to explain these as features with usage instructions.